.github/workflows/deploy.pr.yml

```diff
@@ -1,79 +0,0 @@
-name: Deploy PR
```

Comment:
Do you feel it's easier to maintain all deployments in a single deployment workflow?

Reply:
I thought it's harder and this is why I split them. Even if there's some duplication I think it's easier to debug

Reply:
For me was adding more work load to keep both files synced than executing only the part that I want to test, as this can easily be done when something is not working commenting out the steps you don't need to check.

Also if the "if" logic is working as expected we should be able to know when each flow should run without issues.

Now it's following the same pattern as we have on `safe-react` so should be easier to keep all of them up to date